src/server/routes/entity/work.ts

```diff
@@ -99,26 +99,36 @@ router.get(
 	'/create', auth.isAuthenticated, auth.isAuthorized(ENTITY_EDITOR), middleware.loadIdentifierTypes,
 	middleware.loadLanguages, middleware.loadWorkTypes,
 	middleware.loadRelationshipTypes,
-	(req, res, next) => {
+	async (req, res, next) => {
 		const {Author, Edition} = req.app.locals.orm;
 		let relationshipTypeId;
 		let initialRelationshipIndex = 0;
-		const propsPromise = generateEntityProps(
+		const propsObject = generateEntityProps(
 			'work', req, res, {}
 		);
 
 		if (req.query.author) {
-			propsPromise.author =
-				Author.forge({bbid: req.query.author})
-					.fetch({require: false, withRelated: 'defaultAlias'})
-					.then((data) => data && utils.entityToOption(data.toJSON()));
+			try {
+				const data = await Author.forge({bbid: req.query.author})
+					.fetch({require: false, withRelated: 'defaultAlias'});
+				propsObject.author = utils.entityToOption(data.toJSON());
```

Comment:
I think this will change, with my suggestion above to use the orm function `getEntity`, but for what it's worth compared to the previous implementation you're missing that `data &&` before calling `data.toJSON`.
This was in place because the entity fetch with `required:false` would return null or undefined, and calling `undefined.toJSON()` would throw an error.